cli/cli_mesh_peers.go

```diff
@@ -188,8 +188,12 @@ func peerToOutputString(peer *pb.Peer) string {
 		{Key: "Allows Sending Files", Value: nstrings.GetBoolLabel(peer.IsFileshareAllowed)},
 		{Key: "Accept Fileshare Automatically", Value: nstrings.GetBoolLabel(peer.AlwaysAcceptFiles)},
 	}
+	hostname := peer.Hostname
+	if peer.Nickname != "" {
+		hostname = peer.Nickname
+	}
 	return titledKeyvalListToColoredString(keyval{
-		Key: "Hostname", Value: peer.Hostname,
+		Key: "Hostname", Value: hostname,
```

Comment:
Nickname to be shown instead of full original peer name. How to know original peer name? Will it be shown somewhere?

Reply:
IMO, there should be somewhere shown peer original name to be able to clear nickname in case of something messy gets. Also, IMO, peer should be accessible by nickname and by original name.

Reply:
The peer original name is not displayed anymore, but is still accessible with the autocomplete. I'm thinking to change the meshnet map and display both, nickname and original hostname.

The idea is that a peer can be accessed using 3 names in the app:
* original Nord name
* peer ID
* nickname

The DNS is configured:
* original name, with and without `.nord`
* nickname

Reply:
for the moment changed to display `Nickname [original_host_name]`. I'll update when I'll get more info how to be displayed.